### ISSUE TITLE
Mouse tracking uses text interface even when name should be treated as content

### ISSUE THREAD

**Issue by jcsteh** [nvaccess/nvda#10920]
### Steps to reproduce:

1. In NVDA Settings -> Mouse, enable Mouse tracking and Report role when mouse enters object.
2. Open this URL in Firefox or Chrome:
    `data:text/html,<button aria-label="foo">&nbsp;</button>`
3. Move the mouse to the button.

### Actual behavior:
NVDA says just "button"

### Expected behavior:
NVDA should say "button  foo"

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-19823,5cd0c3fd

#### Windows version:
Version 2004 (OS Build 19041.153)

#### Name and version of other software in use when reproducing the issue:
Firefox 76 nightly
Chrome 83 Canary

### Further info
This occurs because NVDA mouse tracking always uses the text interface where available. It doesn't make sense to do this when the name should be treated as content. We already have this rule in the Gecko vbuf backend and mouse tracking should be consistent.

This could manifest in other ways too. For example:

1. In NVDA Settings -> Mouse, enable Mouse tracking and Report role when mouse enters object.
2. Open this URL in Firefox or Chrome:
    `data:text/html,<button aria-label="foo">bar</button>`
3. Move the mouse to the button.
    - Expected: NVDA should say "button  foo" (label should override)
    - Actual: NVDA says "button  bar"

Originally reported as https://bugzilla.mozilla.org/show_bug.cgi?id=1625254
